README.md

```diff
@@ -1,5 +1,11 @@
 # Percona Monitoring and Management 2.x
-[![Build](https://github.com/percona/pmm/actions/workflows/ci.yml/badge.svg)](https://github.com/percona/pmm/actions/workflows/ci.yml)
+[![Checks](https://github.com/percona/pmm/actions/workflows/common.yml/badge.svg)](https://github.com/percona/pmm/actions/workflows/common.yml)
```

Comment:
I think common.yml should be enough, we have `merge-gatekeeper` there, so if any part of our CI fails we will see a red badge here

Reply:
Ok, let's try this